tests/Unit/Domain/Creators/Test_Brick.cpp

```diff
@@ -101,286 +101,293 @@ void test_brick() {
   const std::vector<std::array<size_t, 3>> refinement_level{{{3, 2, 4}}};
```

Comment:
rename to `Test_Rectilinear.cpp`?

Reply:
We still have `Test_Interval.cpp`, `Test_Rectangle.cpp`, and `Test_Brick.cpp` that haven't changed very much (on purpose, to make sure the domains work the same as before the refactor). Seems fine, no?

Reply:
Ah okay, yea, that's fine then.